src/components/calcite-filter/calcite-filter.tsx

```diff
@@ -99,7 +106,12 @@ export class CalciteFilter {
       <Host>
         <label>
           {this.textLabel}
-          <input type="text" placeholder={this.textPlaceholder} onInput={this.inputHandler} />
+          <input
+            type="text"
+            placeholder={this.textPlaceholder}
+            onInput={this.inputHandler}
+            aria-label={this.textLabel || "Filter"}
```

Comment:
The filter failed accessibility if textLabel was null because there would be no label text. I don't think we want a visible label in all cases (e.g. pickList) so I opted to put a default aria-label and just hardcoded it. 

We can add a separate property if we feel it's valuable to allow this aria label to be overridden when there's not a visible label. I do think its outside the scope of this change though and deserves its own Issue.

Reply:
I think we should add a separate property so that the string can be set without showing visually.

Reply:
Cool. Mind opening an issue for that? This PR is already stacking several changes.

Reply:
Issue created